burstphoto/align.metal

```diff
@@ -416,3 +740,992 @@ kernel void compute_merge_weight(texture2d<float, access::read> texture_diff [[t
     // write weight
     weight_texture.write(weight, gid);
 }
+
+
+// ===========================================================================================================
+// Function specific to merging in the frequency domain
+// ===========================================================================================================
+
+
+kernel void average_y_rgba(texture2d<float, access::read> in_texture [[texture(0)]],
+                           texture1d<float, access::write> out_texture [[texture(1)]],
+                           uint gid [[thread_position_in_grid]]) {
+    uint x = gid;
+    
+    int texture_height = in_texture.get_height();
```

Comment:
Why does this have to be calculated inside the kernel? Can't this be passed as a parameter?

Reply:
I think Metal is extremely efficient in extracting this information from the texture. Still it can be checked in a future version if performance can be increased.

Reply:
This suggestion does not show any improvement of performance.